utils/gpt.py

```diff
@@ -56,17 +58,30 @@ def _make_prompt(self, task: str, prompt_info: dict[str, str] | None) -> str:
     def _send(self, prompt: str, images: list[str]=[], file_path=None) -> str:
         payload = self._create_payload(prompt, images=images)
         if not os.path.exists(file_path):
-            headers = {
-                "Content-Type": "application/json",
-                "Authorization": f"Bearer {api_key}"
-            }
-            # response = requests.post("https://api.anthropic.com/v1/messages", headers=headers, json=payload)  # Anthropic
-            response = requests.post("https://api.gptsapi.net/v1/chat/completions", headers=headers, json=payload)  # WildCard
+            # ANTROPICS
+            if backend == "Claude":
+                headers = {
+                    "Content-Type": "application/json",
+                    "x-api-key": api_key, 
```

Comment:
API key is provided differently than wildcard. This is now handled automatically according to the selected backend.